spacy/displacy/render.py

```diff
@@ -314,6 +314,8 @@ def render_ents(
             label = span["label"]
             start = span["start"]
             end = span["end"]
+            kb_id = str(span.get("kb_id") or "")
+            kb_url = str(span.get("kb_url") or "")
```

Comment:
I am not sure. The get-method for dictionary returns None if the key is not in the dictionary, so `str(None) or ""` returns `'None'`. But we want an empty string if "kb_id" and "kb_url" are not in the dictionary.

Just learned that get-method for dictionary has a second argument, so we can simply write:
```suggestion
            kb_id = span.get("kb_id", "")
            kb_url = span.get("kb_url", "")
```

Reply:
Yep you're right, that's better!